components/transfer/Transfer.vue

```diff
@@ -1,215 +1,274 @@
 <template>
-  <section>
-    <Loader v-model="isLoading" :status="status" />
-    <nuxt-link
-      v-if="$route.query.target"
-      :to="`/${urlPrefix}/u/${correctAddress}`"
-      class="pl-4 is-flex is-align-items-center">
-      <NeoIcon icon="chevron-left" class="mr-2" />
-      {{ $t('teleport.artistProfile') }}
-    </nuxt-link>
-    <p class="title is-size-3">
-      {{ $t('transfer') }} {{ unit }}
-      <span class="has-text-primary">${{ getCurrentTokenValue(unit) }}</span>
-    </p>
-
-    <NeoField>
-      <Auth />
-    </NeoField>
-
-    <div v-if="targets && hasBlockExplorer" class="mb-3">
-      {{ $t('teleport.donationSentTo') }}
-      <a
-        v-for="target in targets"
-        :key="target"
-        :href="addressExplorerUrl(target)"
-        target="_blank"
-        rel="nofollow noopener noreferrer"
-        class="has-text-weight-bold">
-        <Identity ref="identity" :address="target" show-onchain-identity />
-      </a>
+  <div class="transfer-card theme-background-color k-shadow border py-8 px-6">
+    <div
+      class="is-flex is-justify-content-space-between is-align-items-center mb-2">
+      <p class="has-text-weight-bold is-size-3">
+        {{ $t('transfer') }} {{ unit }}
+      </p>
+      <NeoDropdown position="bottom-left" :mobile-modal="false">
+        <template #trigger="{ active }">
+          <NeoButton
+            icon="ellipsis-vertical"
+            class="square-32"
+            :active="active" />
+        </template>
+
+        <NeoDropdownItem
+          v-clipboard:copy="generatePaymentLink(accountId)"
+          @click="toast(`${$i18n.t('toast.urlCopy')}`)">
```

Comment:
just
`$i18n.t('toast.urlCopy')`

doesn't work?

Reply:
Does Not work. There will be a type warning.